src/search/pruning_method.h

```diff
@@ -2,29 +2,35 @@
 #define PRUNING_METHOD_H
 
 #include "operator_id.h"
-#include "task_proxy.h"
+
+#include "../utils/timer.h"
 
 #include <memory>
 #include <vector>
 
 class AbstractTask;
+class State;
+
+namespace limited_pruning {
+class LimitedPruning;
+}
 
 class PruningMethod {
+    utils::Timer timer;
+    friend class limited_pruning::LimitedPruning;
```

Comment:
Why does LimitedPruning need to be a friend class?

Reply:
Good catch; it doesn't need to be.

Reply:
Wait, LimitedPruning was not compiled with the wrong `DEPENDENCY_ONLY` flag noticed by Gabi. Since it directly wants to use the private member `prune_operators` of pruning methods, LImitedPruning has to be a friend class.